### DISCUSSION TITLE
Table grid mode with input boxes

### DISCUSSION THREAD

**Discussion by CrawfordW** [quasarframework/quasar#11561] [General - Components / Directives / etc] [answered]
I'm building a form with a table of inputs. Each row contains several input boxes and the table is quite wide. It would be great if it would switch to grid mode on a small screen, but then the input boxes do not show - just the column names.
Is there a way for grid mode to show components like q-input, or does it only support text content?

**Comment by pdanpdan** [comment, marked as answer]
https://quasar.dev/vue-components/table#example--grid-style-with-slot

Put your form layout in the item slot

**Reply by CrawfordW** [reply]
OK thanks for the tip!
